### ISSUE TITLE
std.deepForce - recursively force all parts of a value

### ISSUE THREAD

**Issue by sbarzowski** [google/jsonnet#779]
It can be useful for:
(a) troubleshooting - it can be temporarily added to simplify the stack traces when some value is used somewhere deeply
(b) optimizing memory usage

A function like this can be already implemented with Jsonnet. One way to do so is to use `assert` to force object fields/array elements and then return the value itself (assert allows us to actually calculate something, but return a completely different thing). It needs to be a builtin for it to really make sense (otherwise both performance and stack traces would probably suffer).

**Comment by sparkprime** [comment]
`assert std.isString(std.toString(foo))` ought to do it

**Comment by sbarzowski** [comment]
Almost - that will fail if there are any functions in there and it will ignore hidden fields of objects.

**Comment by sparkprime** [comment]
true, just need to write something like std.manifestJson that checks the type of everything recursively then (which you need to recurse it anyway).
